src/services/dataService/listItemFunctions.js

```diff
@@ -0,0 +1,41 @@
+const {ListItem} = require('./db')
+
+function findListItem(id) {
+  return ListItem.findOne({
+    where: {id},
+  })
+}
+
+function createListItem(description) {
+  return ListItem.create({
+    description: description
+  })
+}
+
+function deleteListItem(id) {
+  return ListItem.destroy({
+    where: {id},
+  })
+}
+
+function updateListItem(id, data) {
+  return findListItem(id).then(function (listItem) {
+      return listItem.update(data)
+  })
+}
+
+function getListItem (id) {
+  return ListItem.findAll({
```

Comment:
Expecting to return a single instance here, not a collection. So expecting to see `findOne()` rather than `findAll()`.